cypress/e2e/awx/access/teams.cy.ts

```diff
@@ -27,10 +27,10 @@ describe('teams', function () {
     });
   });
 
-  this.afterEach(function () {
+  afterEach(function () {
     cy.deleteAwxUser(user1, { failOnStatusCode: false });
     cy.deleteAwxUser(user2, { failOnStatusCode: false });
-    cy.deleteAwxTeam(team, { failOnStatusCode: false });
+    // cy.deleteAwxTeam(team, { failOnStatusCode: false });
```

Comment:
What is the reason for commenting this out? This could clog the test server with undeleted teams.

Reply:
@vidyanambiar, the reason we commented it out is because a failure is occurring downstream. We need to see whether this fixes it. We had trouble recreating the failure locally, but this is what makes the most sense based on what we were looking at.